wpilibc/src/main/native/cpp/DataLogManager.cpp

```diff
@@ -69,6 +70,11 @@ static std::string MakeLogDir(std::string_view dir) {
       (s.permissions() & fs::perms::others_write) != fs::perms::none) {
     return std::string{usbDir};
   }
+  if (frc::RobotBase::GetRuntimeType() == kRoboRIO) {
```

Comment:
This check shouldn't be required--this entire block is in an ifdef that's only compiled on the Rio.

Reply:
The ifdef doesn't differential roborio 1 vs 2 right?

Reply:
Oh right.  Nevermind.